lib/utilities.js

```diff
@@ -145,6 +145,14 @@ function parseAddress(address) {
 
 module.exports.parseAddress = parseAddress;
 
+// @todo: this "parsing" should be far more rigorous
+module.exports.parseLinkAddress = function(address) {
+  var parts = address.split('/');
```

Comment:
Actually, the defaultSubjects policy might come into play here instead (i.e. don't attempt to segment if we don't cope with subjects). I'll know more once I write the EH integration test.